GabrielaRamirez/GradleExample/workspace/src/main/java/hackkerank/day3/JavaMap.java

```diff
@@ -0,0 +1,33 @@
+package hackkerank.day3;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Scanner;
+
+public class JavaMap {
+
+    public static void main(String[] argh) {
+
+        Scanner in = new Scanner(System.in);
+        int numberOfEntries = in.nextInt();
+        in.nextLine();
+        Map<String, Integer> map = new HashMap<>(numberOfEntries);
```

Comment:
It is recommended that the name of the collections and maps estart with singular name and them the name of the interface, example: numberList

Reply:
Improve the variable name

Reply:
> It is recommended that the name of the collections and maps estart with singular name and them the name of the interface, example: numberList

Could you be more specific please :)